pom.xml

```diff
@@ -384,11 +384,11 @@
                   </group>
                   <group>
                     <title>Example packages</title>
-                    <packages>com.google.cloud.examples*</packages>
+                    <packages>com.google.cloud.examples*:com.google.cloud.nio.examples</packages>
```

Comment:
Should it be examples.*?  It's ok if it should be, examples* just offends my eye if it's not required.